roles/pulibrary.dataspace/vars/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# vars file for roles/pulibrary.oar
```

Comment:
This should read `roles/pulibrary.dataspace`

Reply:
fixed here. https://github.com/pulibrary/princeton_ansible/pull/1668/commits/2701ec473819003eb2f9ad6d8718eda6807b623e